src/components/shared/SearchInput.js

```diff
@@ -24,6 +37,25 @@ class SearchInput extends Component {
     return Svg("magnifying-glass");
   }
 
+  renderArrowButtons() {
+    const { handleNext, handlePrev } = this.props;
+
+    return [
+      arrowBtn(
+        handleNext,
+        "arrow-down",
+        classnames("nav-btn", "next"),
+        "Next Result"
```

Comment:
We might want to add this to `assets/panel/debugger.properties`. You can take a look at #2516 PR to see how tooltip is added